File: 07 - Area of a Rectangular Room/Haskell/main.hs
main :: IO()
area :: Float -> Float -> Float

area l w = l * w

main = do
    print ("What is the length of the room in feet? ")
    l <- readLn :: IO Float
    print ("What is the width of the room in feet? ")
    w <- readLn :: IO Float

    print ("The area is")
    print (show (area l w) ++ " square feet")
    print (show (area l w * 0.09290304) ++ " square metres")

File: 07 - Area of a Rectangular Room/Python/main.py
l = float(input("What is the length of the room in feet? "))
w = float(input("What is the width of the room in feet? "))

print("You entered dimensions of {} feet by {} feet.".format(l, w))
print("The area is\n{} square feet\n{} square metres".format(l*w, l*w*0.09290304))